### DISCUSSION TITLE
Add separate handling in ground items plugin for stackable items

### DISCUSSION THREAD

**Discussion by Spawnova** [runelite/runelite#14906] [Ideas]
Currently there is an option to hide items below a certain value, however I think adding a separate option for stackables would be beneficial.
This would help to declutter the screen a lot during slayer tasks as I often want to pickup stackable runes, coins and noted objects, but not single items worth say 5k or less

Also not sure if this is a bug or a limitation of runelite but duplicate single items on the same tile have their prices added together which will have them appear above the threshold, for example; 3 rune daggers will appear as 12k and show up instead of being 3 seperate items of 4k and being hidden

**Comment by ragsmaroon** [comment]
Was thinking the same thing. I noticed the loot tracker can already recognize noted vs un-noted drops. Would be very beneficial to have ground items be able to recognize this as well as stackable items such as runes like you said. The plugin can already hide/hightlight items under/over a certain quantity, but it's not exactly what's needed in this case.
